web/src/main/java/com/epam/esm/advice/ExceptionControllerAdvice.java

```diff
@@ -1,11 +1,14 @@
 package com.epam.esm.advice;
```

Comment:
I would recommend renaming the 'advice' package.
It's difficult to predict that execution handling placed in the 'advice' package.

Reply:
Commit a89b035